test/deps/binaryen/binaryen_test.cpp

```diff
@@ -84,3 +86,48 @@ TEST(BinaryenTest, Example1) {
   // dispose module
   delete wasm;
```

Comment:
Not really the case of the PR, but still may you remove manual memory management?

```cpp
// parse wast
Module wasm{};
SExpressionParser parser(add_wast.data());
Element &root = *parser.root;
SExpressionWasmBuilder builder(&wasm, *root[0]);

// prepare external interface with imported function's implementation
IntParamExternalInterface interface(env_name, fun_name, fun_impl);

// interpret module
ModuleInstance instance(&wasm, &interface);
```

Reply:
Ok. Not my code, but I can still fix it.